sway-lib-std/src/storage/storage_vec.sw

```diff
@@ -876,6 +876,32 @@ impl<V> StorageKey<StorageVec<V>> {
             }
         }
     }
+
+    /// Clears the `StorageVec`, removing all values.
+    ///
+    /// Note that this method has no effect on the allocated capacity of the vector.
+    ///
+    /// # Examples
+    /// ```sway
+    /// use std::storage::storage_vec::*;
+    ///
+    /// storage {
+    ///     vec: StorageVec<u64> = StorageVec {}
+    /// }
+    ///
+    /// fn foo() {
+    ///     storage.vec.push(1_u64);
```

Comment:
Nitpick. In all the doc examples for all the methods where we have `storage.vec.push` we do not set the type `_u64` explicitly. Therefore, I am for removing it here as well and have simply `storage.vec.push(1)` etc.

I've noticed three places where `let five = 5_u64;` is used. It is not a part of this PR but it's great to have docs consistent so I would go here also for `let five = 5;`.

A nitpick :smile: